### DISCUSSION TITLE
Ground station vs Flight components

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1360] [Q&A]
Hello,

Based on my understanding both Ground station and Satellite uses same Software components.

If it is right, how to differentiate between Flight and Ground station components in Drv and Svc folders ?

Also how to create build for both.

**Comment by LeStarch** [comment]
The above `Svc` and `Drv` are both intended for use in the embedded system (flight components).  The ground system code we use for integration testing is here: https://github.com/fprime-community/fprime-gds
